apps/jamtools/server/src/trace-function.ts

```diff
@@ -0,0 +1,46 @@
+import { trace } from '@opentelemetry/api';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function traceFunction<T extends (...args: unknown[]) => any>(functionName: string, fn: T): T {
```

Comment:
This function wraps any function that ends with `_trace`. An `esbuild` plugin with babel is doing this